### PR TITLE
Download Clublog-"QSLs" to Wavelog

### PR DESCRIPTION
**Pull request by int2001**
This one brings "QSLs" from Clublog to wavelog and adds a cronjob for it to the new Cron-Manager.
Wiki-Entry (for classic users) will follow after merging it.

Includes:
- Taking care of "Clublog-Ignore" for Stations
- Setting Clublog-Ignore to true, if someone tries to download QSLs for Calls he's not granted at Clublog
- Aggregates the Download by "Call" - since Clublog doesn't handle our station_profiles
- Shows QSL-State at QSO-Detail-View

Does not include (to reduce PR, overview, etc.):
- Arrows for Clublog-status (Will do that in a later PR)
- UI for triggering the DL (There's also no UI for manually triggering the upload)

### CODE REVIEW

File: application/migrations/197_clublog_recvd.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				'id' => 'clublog_download',
+				'enabled' => '0',
+				'status' => 'pending',
+				'description' => 'Download QSOs from Clublog',
+				'function' => 'index.php/clublog/download',
+				'expression' => '7 00 * * *',
+				'last_run' => null,
+				'next_run' => null
+			));
+			$this->db->insert_batch('cron', $data);
```
Comment (HB9HIL) [inline application/migrations/197_clublog_recvd.php:24]: please make a check if the entry `clublog_download` already exists.

**Review by HB9HIL — APPROVED** [review]
Tested: 
[x] migration (but please add a check as mentioned above
[x] Cron in Cronmanager
[x] Cron in CLI with Curl
[x] Clublog Ignore
[x] Download with real QSO's
[x] Last Rcvd Date before download

**Review by phl0 — APPROVED** [review]
W0rx!

File: application/migrations/197_clublog_recvd.php (hunk 1 above)
Comment (int2001) [reply application/migrations/197_clublog_recvd.php]: Done. Good find. should be implemented as well at many other points where data is added via migration.

**Pull request merged**